add troubleshooting for mtxs (#1404)
cap-js/docs: get-started/troubleshooting.md

Do not use a `XS_` prefix.

## MTXS

### I get a 401 error when logging in to MTXS through App Router { #mtxs-sidecar-approuter-401}

See [How to configure your App Router](../guides/extensibility/customization#app-router) to verify your setup.

> Also check the [Detailed Documentation about `cds login`](../guides/extensibility/customization#cds-login).

Reviewer: ```suggestion
Also check the [documentation about `cds login`](../guides/extensibility/customization#cds-login).
```

### When running a tenant upgrade, I get the message 'Extensions exist, but extensibility is disabled.'

> This message indicates that extensions exist, but the application is not configured for Extensibility. To avoid accidental data loss by removing existing extensions from the database, the upgrade is blocked in that case.
> Please check the [configuration for Extensibility](../guides/extensibility/customization#_1-enable-extensibility).

Reviewer: ```suggestion
This message indicates that extensions exist, but the application is not configured for extensibility. To avoid accidental data loss by removing existing extensions from the database, the upgrade is blocked in that case.
Please check the [configuration for extensibility](../guides/extensibility/customization#_1-enable-extensibility).
```

Do not use a `XS_` prefix.

## MTXS

### I get a 401 error when logging in to MTXS through App Router { #mtxs-sidecar-approuter-401}

See [How to configure your App Router](../guides/extensibility/customization#app-router) to verify your setup.
Also check the [Detailed Documentation about `cds login`](../guides/extensibility/customization#cds-login).

### When running a tenant upgrade, I get the message 'Extensions exist, but extensibility is disabled.'

This message indicates that extensions exist, but the application is not configured for Extensibility. To avoid accidental data loss by removing existing extensions from the database, the upgrade is blocked in that case.

> Please check the [configuration for Extensibility](http://localhost:5173/docs/guides/extensibility/customization#_1-enable-extensibility).

Reviewer: ```suggestion
Please check the [configuration for Extensibility](../guides/extensibility/customization#_1-enable-extensibility).
```

Author: I had forgotten this great new feature of capire ...

::: danger

> If data loss is intended, you can disable the check by adding
> ```jsonc
> "cds": {
>     "requires": {
>       "cds.xt.DeplyomentService": {
>         "upgrade": {
>           "skipExtensionCheck": true
>         }
>       }
>     }
> }
> ```
> to the configuration.

Reviewer: ```suggestion
If data loss is intended, you can disable the check by adding <Config>cds.requires.cds.xt.DeploymentService.upgrade.skipExtensionCheck = true</Config> to the configuration.
```

Maybe it looks weird, but it's not wrong, or?
![image](https://github.com/user-attachments/assets/99a4b216-e98e-416f-86ca-26fbc4f2f3f0)
